src/terra-dev-site/doc/gettingStarted.a.doc.md

```diff
@@ -0,0 +1,110 @@
+# Getting Started
+
+Terra-dev-site offers a quick site to host test examples and documentation for your react components or md documentation.
+
+## Quick start:
+
+### Installation:
+``` bash
+npm install terra-dev-site
+```
+
+### Pre Reqs:
+* terra-toolkit 3.0.0 or greater
+* react 16
+* react-dom 16
+* README.md in the root project directory
+* Package.json in the root project directory
+* es6 code in src
+* transpiled code in lib
+
+### Running terra-dev-site
+Terra-dev-site automatically gets built when you use the provided webpack config or continently, the tt:serve command provided by terra-toolkit. If you have a custom webpack config, you can either extend the provided webpack config.
+
+Add the following command to your package.json.
+
+```json
+{
+  "scripts": {
+    "start": "tt:serve"
+  }
+}
+```
+
+Then run and navigate to the site in your browser:
+
+```bash
+npm run start
+```
+
+You should see your readme hosted on the site. Next lets add some pages.
+Terra dev site will auto discover files base on site.config.
+
+Add the following file to: `<package>/src/terra-dev-site/doc`
+
+terra.doc.md
+```
+My first page!
+```
+
+Then run and navigate to the site in your browser:
+```bash
+npm run start
+```
+Now you should be able to see your page under
+
+## Pages Auto Discovery
+
+By default terra-dev-site will look for files in the ./src/terra-dev-site folder in dev mode and ./lib/terra-dev-site for prod mode.
```

Comment:
and in an monorepo it will search for files in ./packages/<package>/{src,lib}/terra-dev-site